addons/event_sale/data/event_sale_demo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="event_sale_order_pricelist_1" model="product.pricelist">
```

Comment:
Would not create custom pricelists as demo... this will create a lot side issues. Either use existing stuff from product / sale, either just skip it to keep demo about sales and avoid the "invoicing trap" .